conf/README.md

```diff
@@ -222,6 +222,7 @@ Upgrade related configuration data.
 * `ocp_upgrade_path` - OCP image to upgrade with
 * `ocp_arch` - Architecture type of the OCP image
 * `upgrade_logging_channel` - OCP logging channel to upgrade with
+* `upgrade_ui` - Upgrade from OCS 4.8 to ODF 4.9 via UI (Default: true)
```

Comment:
Please remove Default: true from here as it's not part of default config as well: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml

It's true only when you provide config file ;)

Also please remove the versions from the descriptions, this will be I guess used also for later versions upgrade once it's implemented and it's not worth to mention versions here. 

```suggestion
* `upgrade_ui` - Perform upgrade via UI. (Not all the versions are supported, please look at the code)
```